public/hooks/use_fetch_agentframework_traces.ts

```diff
@@ -33,7 +33,7 @@ export const useFetchAgentFrameworkTraces = (traceId: string) => {
       .catch((error) => dispatch({ type: 'failure', error }));
 
     return () => abortController.abort();
```

Comment:
Shall we add one test case for assert this abort method has been called after unmount.

Reply:
sure, case added.